source/index.html

```diff
@@ -1,12 +1,151 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Седона</title>
+    <link href="css/style.css" rel="stylesheet">
   </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS, уровень 2</a>».</p>
-
+  <body class="page__body">
+    <div class="container">
+      <header class="page-header">
+        <a class="page-header__logo logo" href="">
+        <img class="logo__img" src="" width="" height="" alt="Логотип проект Седона">
+        </a>
+        <nav class="main-nav">
+          <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
+          <div class="main-nav__wrapper">
+            <ul class="main-nav__list site-list reset-list">
+              <li class="site-list__item">
+                <a class="site-list__link" href="#">Главная</a>
+              </li>
+              <li class="site-list__item">
+                <a class="site-list__link" href="catalog.html">Фото и Видео</a>
+              </li>
+              <li class="site-list__item site-list__item--right">
+                <a class="site-list__link" href="form.html">Форма отзыва</a>
+              </li>
+              <li class="site-list__item site-list__item--right">
+                <a class="site-list__link" href="https://htmlacademy.ru/">Html Academy</a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+      </header>
+      <main class="main">
+        <h1 class="visually-hidden">СЕДОНА — небольшой городок в АРИЗОНЕ</h1>
+        <section class="slogan">
+          <h2 class="visually-hidden">Логитин проекта "Седона"</h2>
+          <img class="slogan__img" src="" width="" height=""
+            alt="Welcome to the gorgeous Sedona Because the Grand Canyon sucks!">
+        </section>
+        <section class="description">
+          <div class="description__content">
+            <h2 class="description__title">Седона &ndash; небольшой городок в Аризоне, заслуживающий большего!</h2>
+            <p class="description__sublitle">Рассмотрим 5 причин, по которым седона круче, чем гранд каньон!</p>
```

Comment:
```suggestion
            <p class="description__subtitle">Рассмотрим 5 причин, по которым седона круче, чем гранд каньон!</p>
```

Ты каким редактором пользуешься? Могу тебе подсказать, как поставить туда спеллер, чтобы не допускать орфографических ошибок